pkg/bundle/bundle.go

```diff
@@ -94,13 +95,27 @@ type bundle struct {
 func (b *bundle) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	result, statusPatch, resultErr := b.reconcileBundle(ctx, req)
 	if statusPatch != nil {
-		con, patch, err := ssa_client.GenerateBundleStatusPatch(req.Name, req.Namespace, statusPatch)
+		applyConfig := trustapiac.Bundle(req.Name, req.Namespace).
+			WithStatus(trustapiac.BundleStatus().WithConditions())
```

Comment:
```suggestion
			WithStatus(trustapiac.BundleStatus())
```
Is there a reason you added `.WithConditions()` here?

Reply:
No, thanks - fixed. 😉